### PR TITLE
enable unequal height and width

### PR DESCRIPTION
**Pull request by yahavb**
# What does this PR do?
 enable unequal height and width, currently it is blocked and neuron is going to add that support in the next SDK version.

Fixes # (issue)

## Before submitting
- [x ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ x] Did you make sure to update the documentation with your changes?
- [ x] Did you write any new necessary tests?

### CODE REVIEW

File: optimum/exporters/neuron/model_configs.py [hunk 1]
```diff
@@ -356,12 +357,6 @@ def outputs(self) -> List[str]:
     def generate_dummy_inputs(self, return_tuple: bool = False, **kwargs):
         # For neuron, we use static shape for compiling the unet. Unlike `optimum`, we use the given `height` and `width` instead of the `sample_size`.
```
Comment (JingyaHuang) [inline optimum/exporters/neuron/model_configs.py:358]: Could you remove these two lines of warning as well? Thx!

**Review by JingyaHuang — COMMENTED** [review]
LGTM, thanks for improving it @yahavb ! 

I left a small nit. And could you rebase the branch as well, then I think it's ready to be merged.

File: optimum/exporters/neuron/model_configs.py (hunk 1 above)
Comment (yahavb) [reply optimum/exporters/neuron/model_configs.py]: Yes. removed

**Comment by yahavb** [comment]
Anything else is needed to merge this one?

**Review by JingyaHuang — APPROVED** [review]

**Comment by JingyaHuang** [comment]
Thanks @yahavb for the PR, merged!

**Pull request merged**